assets/model_monitoring/components/src/model_monitor_data_joiner/run.py

```diff
@@ -83,6 +83,12 @@ def run():
         args.right_join_column
     )
 
+    duplicate_column_names = joined_data_df.columns[joined_data_df.columns.duplicated()]
```

Comment:
duplicated() is a build in pyspark function for column list? What does it return? can you give me the API manual for this function? Have you tested this method?